misc/pause-container/Makefile

```diff
@@ -0,0 +1,26 @@
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+.PHONY: all container clean
+
+
+IMAGE = "amazon/amazon-ecs-pause"
+TAG = 1.0
```

Comment:
We're following SemVer 2.0.0 for the ECS agent.  I'd prefer to follow that scheme.